docs/aws.md

```diff
@@ -138,24 +138,22 @@ Nextflow provides built-in support for AWS Batch, allowing the seamless deployme
 
 Read the {ref}`AWS Batch executor <awsbatch-executor>` section to learn more about the `awsbatch` executor in Nextflow.
 
-(aws-batch-config)=
+(aws-batch-cli)=
 
 ### AWS CLI
 
 Nextflow needs the [AWS command line tool](https://aws.amazon.com/cli/) (`aws`) to be available in the container in which tasks are executed, in order to stage input files and output files to and from S3 storage.
 
-:::{tip}
-When using {ref}`wave-page` and {ref}`fusion-page`, the AWS command line tool is not needed for task containers or the underlying EC2 instances when running Nextflow on AWS Batch. See the {ref}`fusion-page` documentation for more details.
-:::
+The `aws` command can be made available in any of the following ways:
```

Comment:
Think Wave tip should stay on top and it should be made clear that installing the `aws` cli  is considered a legacy approach